src/components/Navbar.jsx

```diff
@@ -1,13 +1,114 @@
-import React from "react";
-import { Link } from "react-router-dom";
-import { ABOUT_ROUTE, FAVOURITES_ROUTE, HOME_ROUTE } from "../routes";
+import React, { useState } from "react";
 
 function Navbar() {
+  const [isOpen, setIsOpen] = useState(false);
+
   return (
-    <div>
-      <Link to={HOME_ROUTE}>Home</Link>
-      <Link to={ABOUT_ROUTE}>About</Link>
-      <Link to={FAVOURITES_ROUTE}>Favourites</Link>
+    <div className="w-screen bg-primary items-center md:justify-around justify-between relative z-30">
+      <div
+        className={
+          isOpen
+            ? "drawer drawer-end w-full h-screen absolute top-0"
+            : "drawer drawer-end w-full h-16"
+        }
+      >
+        <input
+          id="my-drawer-3"
+          type="checkbox"
+          onChange={() => setIsOpen(!isOpen)}
+          class="drawer-toggle"
+        />
+        <div class="flex flex-col drawer-content">
+          <div class="w-full navbar md:justify-around justify-between  bg-primary h-16">
+            <div className="flex w-auto items-center md:px-6 px-4">
+              <img className="lg:w-10 w-8" src="./logo.svg" alt="logo" />
+              <h1 className="lg:text-2xl text-xl font-bold">Libvery</h1>
+            </div>
+            <div class="flex-none lg:hidden">
+              <label for="my-drawer-3" class="btn btn-square btn-ghost">
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  fill="none"
+                  viewBox="0 0 24 24"
+                  class="inline-block w-6 h-6 stroke-current"
```

Comment:
There are some errors in the console that related to similar issues like this. You need to change `class=` to `className=` and `stroke-linecap=` to `strokeLinecap=` and so on. You basically have to use camel case.